### ISSUE TITLE
Multisig creation

### ISSUE THREAD

**Issue by nk1tz** [dux-reserve/key-manager#1]
Would also be cool here if I could just supply the Public Key strings plus deriv format path directly as text inputs.
<img width="910" alt="Screen Shot 2021-04-10 at 11 14 22 PM" src="https://user-images.githubusercontent.com/12980165/114296183-a6255580-9a5e-11eb-9711-e62ba7405ccf.png">

**Comment by thibistaken** [comment]
Yes, definitely. It's a feature we would like to add. We could include this as part of a "three dot" advanced menu on the top right corner of the Coldcard box where you could see this option to upload your XPUB info in there.
